app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -95,13 +95,15 @@ private void initView() {
     }
 
     private void renderContributors(List<ContributorViewModel> contributors) {
-        adapter.addAllWithNotify(contributors);
+        adapter.clear();
+        adapter.addAll(contributors);
+        adapter.notifyDataSetChanged();
```

Comment:
It's better to use `DiffUtil` to refresh only changed items 🤔 
But no need to fix on this PR!